benchmarks/perception/a3_stereo_image_proc/launch/launch_sim.launch.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+from ament_index_python.packages import get_package_share_directory
+
+
+from launch import LaunchDescription
+from launch.actions import IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+
+from launch_ros.actions import Node
+
+
+
+def generate_launch_description():
+
+
+    # Include the robot_state_publisher launch file, provided by our own package. Force sim time to be enabled
+    # !!! MAKE SURE YOU SET THE PACKAGE NAME CORRECTLY !!!
+
+    package_name='a3_stereo_image_proc' #<--- CHANGE ME
```

Comment:
Does this need to be changed?

Reply:
nope! thanks for pointing this out! Made the change :)